src/tests/ftest/pool/bad_connect.py

```diff
@@ -75,6 +66,7 @@ def tearDown(self):
             agent_utils.stop_agent(self.agent_sessions)
         server_utils.stop_server(hosts=self.hostlist_servers)
 
+    @skipForTicket("DAOS-3819")
```

Comment:
(this comment applies to unedited lines)

Now that this file inherits `TestWithoutServers`, lines 121-124 can be removed and line 128 can use `self.context`:
```
            pool = DaosPool(self.context)
```